### ISSUE TITLE
jqueryui accordion create event not triggered

### ISSUE THREAD

**Issue by mwyrembl** [BorisMoore/jsviews#412]
When I use the following syntax to create a jqueryui accordion:

```
{^{accordion _create=~accordionCreate _beforeActivate=~accordionBeforeActivate}}
...
{{/accordion}}
```

It only triggers the beforeActivate event but not the create event.
Is there any workaround for this?

**Comment by BorisMoore** [comment]
Ah - yes, that should work. I'll fix it for the next update.

Here is a patched version of the jsviews-jqueryui-widgets.js. Can you test, and confirm it works for you:

[jsviews-jqueryui-widgets.js.txt](https://github.com/BorisMoore/jsviews/files/2780275/jsviews-jqueryui-widgets.js.txt)

If it doesn't then you can use a workaround: Add the following onBind callback:

```
{^{accordion onBind=~onbnd _create=~accordionCreate _beforeActivate=~accordionBeforeActivate}}
...
{{/accordion}}
```

```
onbnd: function() {
    this.initOptions.push("create");
    this.baseApply(arguments);
}
```

**Comment by BorisMoore** [comment]
Fixed in v1.0.2
